src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -72,33 +66,13 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
 
   useExperiences(venue.name);
 
-  const { userWithId } = useUser();
-
   const jazzbarTables = venue.config?.tables ?? JAZZBAR_TABLES;
 
   const [seatedAtTable, setSeatedAtTable] = useState("");
-  const [isAudioEffectDisabled, setIsAudioEffectDisabled] = useState(false);
-
-  const dispatch = useDispatch();
-  const venueId = useVenueId();
-
-  // @debt de-duplicate this with version in src/components/templates/Audience/Audience.tsx
-  const reactionClicked = useCallback(
-    (emojiReaction: EmojiReactionType) => {
-      if (!venueId || !userWithId) return;
-
-      dispatch(
-        addReaction({
-          venueId,
-          reaction: createEmojiReaction(emojiReaction, userWithId),
-        })
-      );
-
-      // @debt Why do we have this here..? We probably shouldn't have it/need it? It's not a very Reacty thing to do..
-      setTimeout(() => (document.activeElement as HTMLElement).blur(), 1000);
-    },
-    [venueId, userWithId, dispatch]
-  );
+
+  const { isShown: isUserAudioOn, toggle: toggleUserAudio } = useShowHide(true);
```

Comment:
Boolean trap. Same as https://github.com/sparkletown/sparkle/pull/1339/files#r667486550

Reply:
Added the OOS + related changes https://github.com/sparkletown/internal-sparkle-issues/issues/912